src/Elmish.WPF/Binding.fs

```diff
@@ -796,89 +796,71 @@ module Binding =
 
 
   module OneWay =
-    /// <summary>
     ///   Elemental instance of a one-way binding.
```

Comment:
I haven't changed the spacing indentation for the content. Let me know if you prefer them to be updated and I will push a change.

Reply:
Yes, let's simply that to just
```fsharp
/// Elemental ...
```